<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Автоаукцион</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>671</width>
      <height>501</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Производитель</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Марка</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Год выпуска</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Цвет</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Пробег</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Начальная цена</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="manufacturerInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>620</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Любой</string>
    </property>
   </widget>
   <widget class="QLabel" name="searchLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Поиск по базе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Найти записи</string>
    </property>
   </widget>
   <widget class="QLabel" name="manufacturerLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Производитель</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="modelLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>650</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Марка</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="modelInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>650</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Любая</string>
    </property>
   </widget>
   <widget class="QLabel" name="yearLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Год выпуска</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="yearInput2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>680</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1886</number>
    </property>
    <property name="maximum">
     <number>3000</number>
    </property>
    <property name="value">
     <number>2050</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="yearInput1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>680</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1886</number>
    </property>
    <property name="maximum">
     <number>3000</number>
    </property>
   </widget>
   <widget class="QLabel" name="yearLabel_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>680</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>—</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="MainLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>База автомобилей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chartButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Диаграмма цен</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disableSearchButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>570</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Сбросить поиск</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorInput">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>620</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Любой</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>620</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Цвет</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mileageLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>650</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пробег</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="priceLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>680</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начальная цена</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="mileageInput1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>650</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="singleStep">
     <number>50000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="mileageLabel_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>650</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>—</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="mileageInput2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>650</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="singleStep">
     <number>50000</number>
    </property>
    <property name="value">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="priceInput1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>680</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
    <property name="singleStep">
     <number>100000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="priceInput2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>680</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
    <property name="singleStep">
     <number>100000</number>
    </property>
    <property name="value">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="priceLabel_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>680</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>—</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>710</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="infoActions">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="guide"/>
    <addaction name="task"/>
    <addaction name="about"/>
   </widget>
   <widget class="QMenu" name="fileActions">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="chooseFile">
     <property name="title">
      <string>Последние файлы</string>
     </property>
    </widget>
    <addaction name="saveFile"/>
    <addaction name="getFile"/>
    <addaction name="chooseFile"/>
    <addaction name="separator"/>
    <addaction name="defaultFile"/>
   </widget>
   <widget class="QMenu" name="tableActions">
    <property name="title">
     <string>Таблица</string>
    </property>
    <addaction name="addElem"/>
    <addaction name="editElem"/>
    <addaction name="delElem"/>
    <addaction name="separator"/>
    <addaction name="cancelElem"/>
   </widget>
   <addaction name="fileActions"/>
   <addaction name="tableActions"/>
   <addaction name="infoActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="guide">
   <property name="text">
    <string>Руководство пользователя</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="task">
   <property name="text">
    <string>Изучить задание</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>Сохранить выборку в файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="getFile">
   <property name="text">
    <string>Открыть файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="addElem">
   <property name="text">
    <string>Добавить запись</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="editElem">
   <property name="text">
    <string>Изменить запись</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="delElem">
   <property name="text">
    <string>Удалить запись(-и)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="cancelElem">
   <property name="text">
    <string>Отменить выделение</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="defaultFile">
   <property name="text">
    <string>Вернуться к стандартной БД</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>chartButton</tabstop>
  <tabstop>manufacturerInput</tabstop>
  <tabstop>modelInput</tabstop>
  <tabstop>yearInput1</tabstop>
  <tabstop>yearInput2</tabstop>
  <tabstop>colorInput</tabstop>
  <tabstop>mileageInput1</tabstop>
  <tabstop>mileageInput2</tabstop>
  <tabstop>priceInput1</tabstop>
  <tabstop>priceInput2</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>disableSearchButton</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
